plugins/PostProcessingPlugin/scripts/Pause_emt.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2019 Ultimaker B.V.
+# The PostProcessingPlugin is released under the terms of the AGPLv3 or higher.
+
+from typing import Optional, Tuple
+
+from UM.Logger import Logger
+from ..Script import Script
+
+class Pause_emt(Script):
+
+    _layer_keyword = ";LAYER:"
+
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name":"Pause (eMotionTech Printers)",
+            "key": "Pause_emt",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "layer_number":
+                {
+                    "label": "Layer",
+                    "description": "At what layer should pause occur. This will be before the layer starts printing.",
+                    "unit": "",
+                    "type": "str",
+                    "default_value": "1"
+                }
+            }
+        }"""
+
+    def execute(self, data: list):
+
+        """data is a list. Each index contains a layer"""
+        layer_nums = self.getSettingValueByKey("layer_number")
+
+        pause = "M600"
```

Comment:
Looking at this code, it looks to me that we have a post-processing plugin that does exactly that (look at https://github.com/Ultimaker/Cura/blob/934248e218c65bc8c4135a465f1c8db37fbd985a/plugins/PostProcessingPlugin/scripts/FilamentChange.py#L76)

Could you explain why do you need this extra post-processing plugin?

Reply:
Hello Diego.
One of your colleague suggested that we upgrade the existing PauseAtHeight plugin script, one of our engineers is working on it... in a couple of hours i'll commit it. 
thanks

Reply:
Hello Diego,
We've been discussing the matter internally and indeed there is no much difference between filament_change and pause_emt. 
So we decided to remove / delete our script (see commit d444b40) and use the existing one...
thanks for your help